tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py

```diff
@@ -0,0 +1,128 @@
+        sandbox_id = dataset_id + '_sandbox'
+        cls.sandbox_id = sandbox_id
+
+        cls.rule_instance = MissingConceptRecordSuppression(
+            project_id, dataset_id, sandbox_id)
+
+        sb_table_names = cls.rule_instance.sandbox_table_for(OBSERVATION)
+
+        cls.fq_sandbox_table_names.append(
+            f'{cls.project_id}.{cls.sandbox_id}.{sb_table_names}')
+
+        cls.fq_table_names = [
+            f'{project_id}.{dataset_id}.{OBSERVATION}',
+            f'{project_id}.{dataset_id}.{CONCEPT}',
+        ]
+        # call super to set up the client, create datasets, and create
+        # empty test tables
+        # NOTE:  does not create empty sandbox tables.
+        super().setUpClass()
+
+    def setUp(self):
+        fq_dataset_name = self.fq_table_names[0].split('.')
+        self.fq_dataset_name = '.'.join(fq_dataset_name[:-1])
+
+        self.date = parser.parse('2020-05-05').date()
+
+        super().setUp()
+
+    def test_missing_concept_record_suppression_cleaning(self):
+        """
+        Tests that concepts missing from vocabulary are suppressed.
+
+        Validates pre conditions, tests execution, and post conditions based on the load
+        statements and the tables_and_counts variable.        
+        """
+        queries = []
+
+        #Append some queries
+
+        create_concepts_query_tmpl = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.concept`
+                (concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, concept_code, valid_start_date, valid_end_date)
+            VALUES
+                (4225432, "Blue color", "Condition", "SNOMED", "Clinical Finding", "84614006", date('2020-05-05'), date('2020-05-05')),
+                (4297377, "Red color" , "Condition" ,"SNOMED" ,"Clinical Finding", "386713009" , date('2020-05-05'), date('2020-05-05')),
+                (441840, "Clinical finding", "Condition", "SNOMED", "Clinical Finding", "404684003", date('2020-05-05'), date('2020-05-05'))
+
+        """).render(fq_dataset_name=self.fq_dataset_name)
+
+        queries.append(create_concepts_query_tmpl)
+
+        create_observations_query_tmpl = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.observation`
+                (observation_id, person_id, observation_concept_id, observation_date, 
+                observation_type_concept_id, value_as_concept_id, qualifier_concept_id, unit_concept_id, 
+                observation_source_concept_id, value_source_concept_id)
+            VALUES
+                (1, 1, 4225432, date('2020-05-05'), 441840, 441840, 441840, 441840, 441840, 441840),
+                (2, 2, 4297377, date('2020-05-05'), 441840, 441840, 441840, 441840, 441840, 441840),
+                (3, 3, 50000000, date('2020-05-05'), 441840, 441840, 441840, 441840, 441840, 441840),
+                (4, 4, 4297377, date('2020-05-05'), 19, 441840, 441840, 441840, 441840, 441840)
```

Comment:
record 3 gets dropped because `observation_concept_id` is bad and 4 gets dropped because `observation_type_concept_id` is bad, right?   Might be a nice jinja comment to add a comment explaining this.  took me a minute to identify why record 4 was dropped.

Reply:
yep, that's correct. Adding comments